core/kernel/nil_spec.rb

```diff
@@ -1,6 +1,16 @@
 require_relative '../../spec_helper'
 require_relative 'fixtures/classes'
 
-describe "Kernel#nil?" do
-  it "needs to be reviewed for spec completeness"
+describe 'Kernel#nil?' do
+  it 'returns false for other than nil' do
```

Comment:
Actually it's simply `returns false`.
There are `NilClass#nil?` specs in https://github.com/ruby/spec/blob/master/core/nil/nil_spec.rb

Reply:
Thank you for reviewing @eregon 

> Actually it's simply returns false.

Sure, I'll update.

> There are NilClass#nil? specs in https://github.com/ruby/spec/blob/master/core/nil/nil_spec.rb

Thanks for the reference, I'll remove the below spec.

Reply:
Made the requested changes. @eregon could you please review?